### ISSUE TITLE
Devportal API list page accessibility issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8390]
### Describe your problem(s)

1.  Select has no accessible name, associated <label>, or [title].

![image](https://user-images.githubusercontent.com/20179540/85012766-218ce200-b181-11ea-8cb1-a0320b5ca79e.png)

2. Duplicate  [id] attributes on the page 

3. `<ul>` without `<li>` eliments

### How will you implement it

---

**Comment by erangatl** [comment]
Related: https://github.com/wso2/product-apim/issues/8342,
